sql/core/src/test/scala/org/apache/spark/sql/execution/SparkSqlParserSuite.scala

```diff
@@ -259,4 +259,22 @@ class SparkSqlParserSuite extends AnalysisTest {
       parser.parsePlan("ALTER SCHEMA foo SET DBPROPERTIES ('x' = 'y')"))
     assertEqual("DESC DATABASE foo", parser.parsePlan("DESC SCHEMA foo"))
   }
+
+  test("manage resources") {
+    assertEqual("ADD FILE abc.txt", AddFileCommand("abc.txt"))
+    assertEqual("ADD FILE \'abc.txt\'", AddFileCommand("abc.txt"))
+    assertEqual("ADD FILE \"/path/to/abc.txt\"", AddFileCommand("/path/to/abc.txt"))
+    assertEqual("LIST FILE abc.txt", ListFilesCommand("abc.txt".split("\\s+")))
```

Comment:
Shall we use simply `Array("/path//abc.txt")` instead of `"abc.txt".split("\\s+")`? (here and the below `LIST` test cases)

Reply:
List commands are using `.split("\\s+")` on path given to them. Thus we have to do same here.

Reply:
No. This is a test case. You can use the result of `.split("\\s+")`.

Reply:
Ok. I will use the result of .split("\\s+").